src/main/resources/assets/neoforge/lang/en_us.json

```diff
@@ -147,51 +147,15 @@
   "neoforge.configuration.uitext.title.common": "%s Common Configuration",
   "neoforge.configuration.uitext.title.startup": "%s Startup Configuration",
 
-  "neoforge.configuration.uitext.notonline": [
-    {
-      "text": "Settings in here are determined by the server and cannot be changed while online.",
-      "color": "red"
-    }
-  ],
-  "neoforge.configuration.uitext.notlan": [
-    {
-      "text": "Settings in here cannot be edited while your game is open to LAN. Please return to the main menu and load the world again.",
-      "color": "red"
-    }
-  ],
-  "neoforge.configuration.uitext.notloaded": [
-    {
-      "text": "Settings in here are only available while a world is loaded.",
-      "color": "red"
-    }
-  ],
-  "neoforge.configuration.uitext.unsupportedelement": [
-    {
-      "text": "This value cannot be edited in the UI. Please contact the mod author about providing a custom UI for it.",
-      "color": "red"
-    }
-  ],
-  "neoforge.configuration.uitext.longstring": [
-    {
-      "text": "This value is too long to be edited in the UI. Please edit it in the config file.",
-      "color": "red"
-    }
-  ],
+  "neoforge.configuration.uitext.notonline": "Settings in here are determined by the server and cannot be changed while online.",
+  "neoforge.configuration.uitext.notlan": "Settings in here cannot be edited while your game is open to LAN. Please return to the main menu and load the world again.",
+  "neoforge.configuration.uitext.notloaded": "Settings in here are only available while a world is loaded.",
+  "neoforge.configuration.uitext.unsupportedelement": "This value cannot be edited in the UI. Please contact the mod author about providing a custom UI for it.",
+  "neoforge.configuration.uitext.longstring": "This value is too long to be edited in the UI. Please edit it in the config file.",
   "neoforge.configuration.uitext.section": "%s...",
   "neoforge.configuration.uitext.sectiontext": "Edit",
-  "neoforge.configuration.uitext.breadcrumb": [
-    {
-      "index": 0
-    },
-    {
-      "text": " > ",
-      "color": "gold",
-      "bold": true
-    },
-    {
-      "index": 1
-    }
-  ],
+  "neoforge.configuration.uitext.breadcrumb.order": "%1$s%2$s%3$s",
+  "neoforge.configuration.uitext.breadcrumb.seperator": " > ",
```

Comment:
Should we perhaps combine these two into a single translation key? Having the two be separated doesn't sit right with me, and seems a bit too heavy a burden to understand for a translator.

It should be possible to combine them, and still have the separator be colored. Based on my testing, you'd have to apply the gold and boldface formatting on the combined breadcrumb translation, and explicitly set no-boldface and white formatting on the individual titles. (When composing the breadcrumb translation, only the 'child' or second parameter is set to explicit no-boldface and white, assuming the parent is already set as such.)

Here's some example code of what I mean:
```java
Style titleStyle = Style.EMPTY.withColor(CommonColors.WHITE).withBold(false);
Style crumbStyle = Style.EMPTY.withColor(ChatFormatting.GOLD).withBold(true);
String crumb = "%1$s > %2$s";
//String crumb = "%2$s < %1$s";

List<String> trail = List.of("Apple", "Banana", "Cherry", "Durian");

MutableComponent component = Component.literal(trail.getFirst()).withStyle(titleStyle);
for (String s : trail.subList(1, trail.size())) {
    component = Component.translatable(crumb,
            component,
            Component.literal(s).withStyle(titleStyle)
    ).withStyle(crumbStyle);
}
```

If that isn't acceptable, then at least move the spaces from the `.separator` to the `.order` to visibly separate the three positional parameters.

Reply:
I'm not a fan of forcing a hardcoded copy of the default onto elements. Such stuff tends to come apart at the seams with time...